src/flows/components/FlowContextMenu.tsx

```diff
@@ -16,27 +16,47 @@ interface Props {
 }
 
 const FlowContextMenu: FC<Props> = ({id, name}) => {
-  const {remove} = useContext(FlowListContext)
+  const {remove, clone} = useContext(FlowListContext)
 
   const handleDelete = () => {
     event('delete_notebook')
     remove(id)
   }
 
+  const handleClone = () => {
+    event('clone_notebook')
+    clone(id)
```

Comment:
would we want to redirect to the cloned notebook?

Reply:
Can do. Shouldn't that be in the actual action and not this clone handler?

Reply:
tough call. i think we'd always want to go to the clone given the few examples i've heard people talk about, but having redirects at the action layer has bitten us in the past on these sorts of things

i'd keep it on the view layer here in case we want to decouple routing from asset generation

Reply:
Fiar enough